.vscode/extensions.json

```diff
@@ -1,5 +1,6 @@
 {
     "recommendations": [
         "davidanson.vscode-markdownlint",
```

Comment:
Do we still need the `markdownlint` extension? Isn't it now superseded with Prettier?

Reply:
Not completely. We still keep it around because it does some checks prettier doesn't. I'm not sure it's worth it, but for now it stays.